src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -142,15 +120,33 @@ def assertCbsdRecord(self, registration_request, grant_request, grant_response,
           self.assertEqualToDeviceOrPreloadedConditionalParam('indoorDeployment', \
               device['installationParam'], reg_conditional_device_data['installationParam'],\
                 cbsd_record[0]['installationParam'])
-          #  parameters should equal to device or to default value            
-          self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaAzimuth', \
-              device['installationParam'], reg_conditional_device_data['installationParam'],\
-                cbsd_record[0]['installationParam'], 0)
-                
-          self.assertEqualToDeviceOrPreloadedConditionalParamOrDefaultValue('antennaBeamwidth', \
-              device['installationParam'], reg_conditional_device_data['installationParam'],\
-                cbsd_record[0]['installationParam'], 360)
-            
+          #  parameters should equal to device or to default value
+          registered_antenna_azimuth = device['installationParam']['antennaAzimuth'] \
+            if 'antennaAzimuth' in device['installationParam'] \
+            else reg_conditional_device_data['installationParam']['antennaAzimuth'] \
+            if ['antennaAzimuth'] in reg_conditional_device_data['installationParam'] else None
+          registered_antenna_beamwidth = device['installationParam']['antennaBeamwidth'] \
+            if 'antennaBeamwidth' in device['installationParam'] \
+            else reg_conditional_device_data['installationParam']['antennaBeamwidth'] \
+            if ['antennaBeamwidth'] in reg_conditional_device_data['installationParam'] else None
+          dump_antenna_beamwidth = cbsd_record[0]['installationParam']['antennaBeamwidth'] \
+            if 'antennaBeamwidth' in cbsd_record[0]['installationParam'] else None
+          dump_antenna_azimuth = cbsd_record[0]['installationParam']['antennaAzimuth'] \
+            if 'antennaAzimuth' in cbsd_record[0]['installationParam'] else None
+          is_default_dump_beamwidth = dump_antenna_beamwidth == 360 or dump_antenna_beamwidth == 0
+          is_default_dump_azimuth = dump_antenna_azimuth == 360 or dump_antenna_azimuth == 0
```

Comment:
This doesn't make sense as there is no default value for azimuth (and you don't use it anyway).

Reply:
I already have deleted it in the commit : 
# remove unnecessary variable  9b19072
I don't know why it didn't appear in the cod :/

Reply:
Who knows... the GitHub interface is far from ideal, so maybe I displayed the wrong diff or interpreted it incorrectly. Anyway, glad it's fixed :)